### PR TITLE
Read The Docs

### LINKED ISSUE #89: documentation
I'd like to start creating a documentation website... I think the doc strings can be easily imported into readthedocs or something similar.

Assign this one to me.

### PR DESCRIPTION
**Pull request by gonsie**
Basic documentation for read the docs.

Closes #89

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# SimManager Documentation
```
Comment (dinatale2) [inline docs/README.md]: Did you mean `MaestroWF` instead of `SimManager` throughout this document?

Comment (FrankD412) [reply docs/README.md]: The full name is `Maestro Workflow Conductor` -- if that should be the official name.

**Comment by FrankD412** [comment]
@gonsie -- This might be a silly question... how do I view the docs? I seem to get to a page for the modules that says the page is missing. Should I just build them locally?

**Comment by gonsie** [comment]
@FrankD412 You can build them locally with `make html` in the `docs/` directory. Then open the file `build/html/index.html`.

**Pull request merged**